wrapanapi/systems/nuage.py

```diff
@@ -62,7 +62,9 @@ def list_network_groups(self):
         return self._request_list('/enterprises', 'get')
 
     def list_cloud_subnets(self):
-        return self._request_list('/subnets', 'get', exclude_name='BackHaulSubnet')
+        subnets = self._request_list('/subnets', 'get', exclude_name='BackHaulSubnet')
+        l2domains = self._request_list('/l2domains', 'get')
```

Comment:
How can the caller tell these different resource types apart?  This seems like it should be a separate method, or that this should return a dict with 'subnet' and 'l2domain' keys.

Reply:
Thanks for noticing @mshriver, you are right. I provided two separate calls now, please see https://github.com/ManageIQ/wrapanapi/pull/326. Note, however, that Nuage provider inventories both 'subnet' and 'l2domain' as CloudSubnet hence the count on provider details page is in fact a sum of the two.